lib/client/identityfile/identity.go

```diff
@@ -150,6 +150,9 @@ type WriteConfig struct {
 	KubeProxyAddr string
 	// KubeTLSServerName is the SNI host value passed to the server.
 	KubeTLSServerName string
+	// KubeStoreAllHostCAs stores the CAs of all clusters in kubeconfig, instead
+	// of just the root cluster's CA.
+	KubeStoreAllHostCAs bool
```

Comment:
Nit: Technically, those are not "host" CAs. Host CA is the one that signs SSH certificates for nodes.
```suggestion
	KubeStoreAllCAs bool
```